src/DotNetSdkHelpers/Helpers.cs

```diff
@@ -18,9 +18,9 @@ public static class Helpers
                         "https://raw.githubusercontent.com/dotnet/core/master/release-notes/releases-index.json"))
                 .Releases;
 
-        public static async Task<List<Release>> GetReleases(Uri address) =>
-            JsonConvert.DeserializeObject<ReleasesResponse>(
-                    await Client.GetStringAsync(address))
+        public static async Task UpdateReleases(this ReleaseChannel channel) =>
```

Comment:
This was changed so I could more easily correlate a channel with its child releases. There was a whole async Lazy<T> issue to solve here and it just seemed easier to do this than to make it more complicated.